### DISCUSSION TITLE
More languages

### DISCUSSION THREAD

**Discussion by wardachD** [ME-MarvinE/XCalendar#121] [Ideas]
Hello! First of all, good job with the calendar! What would be good imo is to add the possibility to change the language (of days) or translate them oneself. 
Unless it's already possible?

**Comment by ME-MarvinE** [comment]
If you mean the days of the week, you can use the CalendarView's `DayNameTemplate` property to add your own translation.

If you mean the numbers representing the day, you can use the CalendarView's `DayTemplate` property to add your own translation. You can use the `DayView` control to replicate default functionality.

Both of these methods are used in the examples found in the sample app.

Similar questions to yours: #29, #42

**Reply by ME-MarvinE** [reply]
Automatic translations are now also possible by changing `CultureInfo.CurrentCulture` and `CultureInfo.CurrentUICulture` on initialization in App.xaml.cs. More info in the wiki.
